resources/gui.py

```diff
@@ -485,7 +485,8 @@ def place_colony(self, instance, touch):
             with self.simulation_widget.canvas:
                 Image(source="../images/colony.png", pos=(transformed_touch[0] - 50, transformed_touch[1] - 50), size=(100, 100))
             self.simulation_widget.unbind(on_touch_down=self.place_colony)
-            sim.add_colony(Colony(amount=100, size=(100, 100), coordinates=(transformed_touch[0] - 50, transformed_touch[1] - 50), color=(0, 0, 0, 1)))
+            sim.add_colony(Colony(grid_pheromone_shape=(100, 100), amount=100, size=(100, 100),
```

Comment:
Why did you use `grid_pheromone_shape=(100, 100)`? Shouldn't it be `grid_pheromone_shape=(int(sim.bounds[3]-sim.bounds[2]), int(sim.bounds[1]-sim.bounds[0]))` ? Or what did i miss here?

Reply:
`grid_pheromone_shape` is a parameter that is chosen independently of the size of the board (bounds). This parameter will be experimented with to find the best value for it, so that the ants can create trails and locate and transport their food as quickly as possible.

Example parameter values:
```
n_row, n_col = int(sim.bounds[3]-sim.bounds[2]), int(sim.bounds[1]-sim.bounds[0])


grid_pheromone_shape=(n_row, n_col) #width_spot, height_spot = 1, 1
grid_pheromone_shape=(n_row*2, n_col*2) #width_spot, height_spot = 0.5, 0.5
```
So, it's not initially important what value `grid_pheromone_shape` should be assigned.

![pheromone_array](https://github.com/heyitsalina/ant_search_algorithm/assets/147917604/805e2d54-0956-4a26-ac0b-996dbce2bf60)

It would be advantageous to obtain the value for this parameter as input through a GUI (Colony Information window). What do you think @Jonah-gr ? If you agree with this, then I would create an issue regarding this with low priority.

Reply:
Ok, then we can leave it as it is for now. I also think that the array should not be as big as the canvas. A setting option via the GUI should be possible!